website/public/policy-templates/windows/firezone.admx

```diff
@@ -0,0 +1,214 @@
+    >
+      <parentCategory ref="firezone" />
+      <supportedOn ref="SUPPORTED_FZ_GUI_1_4_15" />
+      <elements>
+        <text id="logFilter" required="true" />
+      </elements>
+    </policy>
+
+    <policy
+      name="accountSlug"
+      class="User"
+      displayName="$(string.accountSlug)"
+      explainText="$(string.accountSlug_explain)"
+      key="Software\Policies\Firezone"
+      presentation="$(presentation.accountSlug)"
+      valueName="accountSlug"
+    >
+      <parentCategory ref="firezone" />
+      <supportedOn ref="SUPPORTED_FZ_GUI_1_4_15" />
+      <elements>
+        <text id="accountSlug" required="true" />
+      </elements>
+    </policy>
+
+    <policy
+      name="internetResourceEnabled"
+      class="User"
+      displayName="$(string.internetResourceEnabled)"
+      explainText="$(string.internetResourceEnabled_explain)"
+      key="Software\Policies\Firezone"
+      presentation="$(presentation.internetResourceEnabled)"
+      valueName="internetResourceEnabled"
+    >
+      <parentCategory ref="firezone" />
+      <supportedOn ref="SUPPORTED_FZ_GUI_1_4_15" />
+      <elements>
+        <boolean id="internetResourceEnabled">
+          <trueValue>
+            <decimal value="1" />
+          </trueValue>
+          <falseValue>
+            <decimal value="0" />
+          </falseValue>
+        </boolean>
+      </elements>
+    </policy>
+
+    <policy
+      name="hideAdminPortalMenuItem"
+      class="User"
+      displayName="$(string.hideAdminPortalMenuItem)"
+      explainText="$(string.hideAdminPortalMenuItem_explain)"
+      key="Software\Policies\Firezone"
+      presentation="$(presentation.hideAdminPortalMenuItem)"
+      valueName="hideAdminPortalMenuItem"
+    >
+      <parentCategory ref="firezone" />
+      <supportedOn ref="SUPPORTED_FZ_GUI_1_4_15" />
+      <elements>
+        <boolean id="hideAdminPortalMenuItem">
+          <trueValue>
+            <decimal value="1" />
+          </trueValue>
+          <falseValue>
+            <decimal value="0" />
+          </falseValue>
+        </boolean>
+      </elements>
+    </policy>
+
+    <policy
+      name="connectOnStart"
+      class="User"
+      displayName="$(string.connectOnStart)"
+      explainText="$(string.connectOnStart_explain)"
+      key="Software\Policies\Firezone"
+      presentation="$(presentation.connectOnStart)"
+      valueName="connectOnStart"
+    >
+      <parentCategory ref="firezone" />
+      <supportedOn ref="SUPPORTED_FZ_GUI_1_4_15" />
+      <elements>
+        <boolean id="connectOnStart">
+          <trueValue>
+            <decimal value="1" />
+          </trueValue>
+          <falseValue>
+            <decimal value="0" />
+          </falseValue>
+        </boolean>
+      </elements>
+    </policy>
+
+    <policy
+      name="startOnLogin"
+      class="User"
+      displayName="$(string.startOnLogin)"
+      explainText="$(string.startOnLogin_explain)"
+      key="Software\Policies\Firezone"
```

Comment:
I think we may also be able to directly write to `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run` which manages start-up applications. I am not sure if that is a good idea though so for now, I've implemented it in the code.

Reply:
In the current solution, Firezone needs to be started at least once while this setting is enabled for it to set itself to autostart on login.

Reply:
Same limitation in macOS.

There, admins typically manage startup items using a different payload targeted to one of Apple's daemons and which can be locked against disabling by the user.

Windows should have something similar so we probably just need to document it for users.

Reply:
> Oh I just saw your previous comment.

Well, `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run` is still writeable by the current user, even without admin privileges according to my research.